From bc503b32060e7fa0ad11b7fd70fac2614241af9b Mon Sep 17 00:00:00 2001
From: Eric Long <i@hack3r.moe>
Date: Wed, 11 Oct 2023 21:41:25 +0800
Subject: [PATCH] Lock stb to last revision with stb_image_resize.h

Newer revisions of nothings/stb have switched to stb_image_resize2.h and
moved the original header to deprecated/stb_image_resize.h since [1].
Pin down the version to prevent future breaking changes.

[1]: https://github.com/nothings/stb/commit/c4bbb6e75f688318b2df2b70c2df2d641c1a8481
---
 subprojects/stb.wrap | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/subprojects/stb.wrap b/subprojects/stb.wrap
index 27bdb35b3..0083f4f76 100644
--- a/subprojects/stb.wrap
+++ b/subprojects/stb.wrap
@@ -2,7 +2,7 @@
 directory = stb
 
 url = https://github.com/nothings/stb.git
-revision = master
+revision = 5736b15f7ea0ffb08dd38af21067c314d6a3aae9
 depth = 1
 
 patch_directory = stb
